apps/crn-server/src/utils/urls.ts

```diff
@@ -1,4 +1,8 @@
+import buildQuery, { Filter } from 'odata-query';
 import { createUrlFactory } from '@asap-hub/squidex';
 import { appName, baseUrl } from '../config';
 
 export const createUrl = createUrlFactory({ appName, baseUrl });
+
+export const buildODataFilter = (filterObj: Filter): string =>
```

Comment:
isnt this the same as https://github.com/yldio/asap-hub/blob/18bc42017ae19b6f777075404e3fc0e0de5fb192/apps/crn-server/src/utils/odata.ts#L29

Reply:
still outstanding